float32 diffTachoLeft
float32 diffTachoRight
---
geometry_msgs/Pose diffPosition